asv_bench/benchmarks/frame_methods.py

```diff
@@ -322,9 +322,9 @@ class Dropna:
     def setup(self, how, axis):
         self.df = DataFrame(np.random.randn(10000, 1000))
         with warnings.catch_warnings(record=True):
-            self.df.ix[50:1000, 20:50] = np.nan
```

Comment:
Should the with-statement not be removed and this block be dedented?

Reply:
i think you're right, yes